### ISSUE TITLE
Ship Fittings/Loadout doesn't show scripted Modules correctly - EFT Export problem

### ISSUE THREAD

**Issue by elvira-hoelzl** [GoldenGnu/jeveassets#498]
In Assets you can see the modules and scripts:
```
Tracking Computer II	None	Tracking Computer	Module	Medium	Small	... 1	Medium Slot 3
Optimal Range Script	None	Tracking Script	Charge	None	Small	...	1	Medium Slot 3
Sensor Booster II	None	Sensor Booster	Module	Medium	Small  ...	1	Medium Slot 2
Scan Resolution Script	None	Sensor Booster Script	Charge	None	Small ...	1	Medium Slot 2
```
But in Ship Fittings the scripts are listed like they where modules:
```
- Medium Slots
Sensor Booster II	1,096,000.00 isk
Coreli A-Type 5MN Microwarpdrive	47,080,000.00 isk
Tracking Computer II	1,973,000.00 isk
Optimal Range Script	19,590.00 isk
Scan Resolution Script	24,630.00 isk
Coreli A-Type 1MN Afterburner	47,100,000.00 isk
```

And EFT Export now misses the actual modules and only shows the scripts (and thus is not importable correctly anymore).
```
Coreli A-Type 1MN Afterburner
Scan Resolution Script
Optimal Range Script
Coreli A-Type 5MN Microwarpdrive
```

Maybe ignoring charges for the Ship Fittings would be a solution or moving them to cargo (so at least the information they are there is not lost - in case somebody generates a buy-all list).

**Comment by coanghel** [comment]
Will take a look at this one

**Comment by GoldenGnu** [comment]
Released in 7.9.3
